example_project_bzl_4/.bazelrc

```diff
@@ -0,0 +1,2 @@
+try-import %workspace%/.output-bazelrc
+build --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host
```

Comment:
Wait, we need `build --incompatible_enable_cc_toolchain_resolution` if we're going to use platforms resolution.

Reply:
`--incompatible_enable_cc_toolchain_resolution` only affects cc_rules.
Afaik, the platform resolution as a Bazel mechanism has been turned on, by default, for a while now - in this particular case, we are using it to inform `rules_nixpkgs` that the current host has nix installed. 

However, using platforms in the example C++ code is very good idea - I would suggest to implement it in subsequent prs.